arch/arm/src/stm32h5/Kconfig

```diff
@@ -52,7 +53,13 @@ config STM32H5_STM32H563XX
 	# STM32H552 and STM32H562 devices documented in RM0439
 	bool
 	default n
-	
+
+# This option was added because Ethernet is only supported on H563/573 chips
+config STM32H5_STM32H5X3XX
```

Comment:
@stbenn hmm, but H523 and H533 will fit this mask, and some user could enable it thinking it was the right thing. Maybe it is better to select STM32H5_HAVE_ETHERNET directly at STM32H5_STM32H563XX and STM32H5_STM32H573XX. What do you think?

Reply:
I agree. I will make update it to be exclusively h563/h573.